nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml

```diff
@@ -5,6 +5,9 @@ on: [push, pull_request]
 
 jobs:
   test:
+    env: 
+      NXF_VER: {% raw %}${{ matrix.nxf_ver }}{% endraw %}
```

Comment:
I think we can get rid of the `raw` stuff and the `$` symbol here..

Just `NXF_VER: {{ matrix.nxf_ver }}` should be fine I think.

Reply:
Thanks, corrected that now